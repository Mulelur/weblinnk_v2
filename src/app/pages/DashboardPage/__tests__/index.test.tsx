import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { DashboardPage } from '..';

const shallowRenderer = createRenderer();

describe('<DashboardPage />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<DashboardPage />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
