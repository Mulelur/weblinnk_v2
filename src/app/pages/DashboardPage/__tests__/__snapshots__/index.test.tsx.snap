// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardPage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
    prioritizeSeoTags={false}
  >
    <title>
      Dashboard Page
    </title>
    <meta
      content="A React Boilerplate application homepage"
      name="description"
    />
  </Helmet>
  <PageWrapper>
    <NavBar />
    <DashboardPage__Container>
      <Projects />
    </DashboardPage__Container>
  </PageWrapper>
</React.Fragment>
`;
